### PR TITLE
Adding Docs 

### PR DESCRIPTION
**Pull request by BradReesWork**
creating docs

### CODE REVIEW

**Comment by dongxuy04** [comment]
LGTM, thanks! I think we can merge this and I will add more doc pages to the repo.

**Review by dongxuy04 — APPROVED** [review]
LGTM, thanks! I think we can merge this and I will add more doc pages to the repo.

File: docs/wholegraph/source/installation/source_build.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+## Prerequisites
+
+__Compiler__:
+* `gcc`         version 11.0+
+* `nvcc`        version 11.0+
+* `cmake`       version 3.23.1+
+
+__CUDA:__
+* CUDA 11.8+
```
Comment (acostadon) [inline docs/wholegraph/source/installation/source_build.md:17]: Is it CUDA 11.8 ?

**Review by acostadon — APPROVED** [review]
I wasn't able to build the docs but the changes look good as far as I can tell.

**Pull request merged**